test/cookbooks/selinux_policy_test/recipes/fcontext.rb

```diff
@@ -40,7 +52,6 @@
   notifies :run, 'ruby_block[fail-mismatch]', :immediate
 end
 
-# Should not run again
```

Comment:
As a thought, please add a separate test recipe for this so it won't interfere with this one